docsite.html

```diff
@@ -429,15 +423,138 @@ <h3>Badges</h3>
+
+                    <!-- Extra small avatar  -->
+
+                    <img class="avatar-xsm img-circle" src="http://tiny.cc/2n7ouz" alt="">
+
+
+
+                </div>
+
+                <p class="small-text mt-24">Usage and code-snippet</p>
+                <p class='x-small-text mt-24'>To use the above shown avatars : You just have to copy the code snippet
+                    below and paste it in your html file. Some of the styles here are :<br> </p>
+                <ul class='x-small-text p-16'>
+                    <li>Large avatar</li>
+                    <li>Medium avatar</li>
+                    <li>Small avatar</li>
+                    <li>Extra small avatar</li>
+
+
+                </ul>
+
+
+                <div class="mt-24">
+                    <code>
+                        <iframe class="code-snippet"
+                            src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=one-light&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=47px&ph=30px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253C%21--%2520Large%2520avatar%2520%2520--%253E%250A%250A%253Cimg%2520class%253D%2522img-lg%2520img-circle%2522%2520src%253D%2522http%253A%252F%252Ftiny.cc%252F2n7ouz%2522%2520alt%253D%2522%2522%253E%250A%250A%253C%21--%2520Medium%2520avatar%2520%2520--%253E%250A%250A%253Cimg%2520class%253D%2522avatar-md%2520img-circle%2522%2520src%253D%2522http%253A%252F%252Ftiny.cc%252F2n7ouz%2522%2520alt%253D%2522%2522%253E%250A%250A%253C%21--%2520Small%2520avatar%2520%2520--%253E%250A%250A%253Cimg%2520class%253D%2522avatar-sm%2520img-circle%2522%2520src%253D%2522http%253A%252F%252Ftiny.cc%252F2n7ouz%2522%2520alt%253D%2522%2522%253E%250A%250A%253C%21--%2520Extra%2520small%2520avatar%2520%2520--%253E%250A%250A%253Cimg%2520class%253D%2522avatar-xsm%2520img-circle%2522%2520src%253D%2522http%253A%252F%252Ftiny.cc%252F2n7ouz%2522%2520alt%253D%2522%2522%253E"
+                            style="border:0; transform: scale(1); overflow:hidden;"
                             sandbox="allow-scripts allow-same-origin">
                         </iframe>
                     </code>
 
                 </div>
 
             </section>
+            <section class="component-section-wrapper p-40 " id='images'>
+                <h3>Image</h3>
+                <p class='x-small-text mt-24'>Images are an important part of the web. Images are basically
+                    storytellers, they can complement the content on your web-application.</p>
+
+                <p class="small-text mt-24">Responsive image</p>
+                <p class='x-small-text mt-24'>The image given below can adjust its size according to the container while
+                    maintaining its aspect ratio.</p>
+
+
+                <div class="component-wrapper p-16 br-m mt-24">
+                    <!-- Responsive image  -->
+                    <img class='resp-img'' src="http://tiny.cc/nm7ouz" alt="">
```

Comment:
Inverted commas of class are not properly closed

Reply:
Corrected it, thank you.